csrc/tensor_view.cpp

```diff
@@ -565,6 +565,21 @@ TensorView* TensorView::merge(int64_t axis_o, int64_t axis_i) {
   return this;
 }
 
+TensorView* TensorView::flatten(int64_t from, int64_t to) {
+  NVF_ERROR(nDims() > 0, "Tried to do flatten on a 0-dim TensorView");
+  from = wrapDim(from);
+  to = wrapDim(to);
+  if (from > to) {
+    std::swap(from, to);
+  }
```

Comment:
This seems to be unnecessarily permissive. Does torch do this?

Reply:
No PyTorch doesn't. I don't have any use case that uses this "feature". I can remove it here and if it turns out to be inconvenient, I can add it back later.